lib/string.gd

```diff
@@ -836,4 +836,13 @@ BindGlobal("BHINT", MakeImm("\>\<"));
 
 #############################################################################
 ##
+#F MemoryString( <m> )    returns an appropriate human-readable string 
+##                        representation of <m> bytes
```

Comment:
If you asked me what a function named "MemoryString" would do, I am not sure I'd ever have guessed this meaning...
Alas, I don't really have a better idea for a name :/